maintainers/maintainer-list.nix

```diff
@@ -8601,6 +8601,12 @@
     githubId = 19472270;
     name = "Sebastian";
   };
+  sebbadk = {
```

Comment:
Please put this into a separate commit with the message ``maintainers: add sebbadk``.